packages/components/src/Button/Button.css

```diff
@@ -48,12 +58,12 @@ a.button:active {
   background-color: var(--button--color-surface--hover);
 }
 
-.button:focus,
-a.button:focus,
+.button:focus-visible,
+a.button:focus-visible,
 .button:active,
 a.button:active {
   box-shadow: var(--shadow-focus);
-  outline: none;
+  outline: transparent;
```

Comment:
Allows users with high-contrast assistive tech to override our outline with a more powerful one of their choosing